backend/migrations/versions/458572580709_remove_dataset_table_read_permissions_from_env_admins.py

```diff
@@ -0,0 +1,87 @@
+"""remove_dataset_table_read_permissions_from_env_admins
+
+Revision ID: 458572580709
+Revises: c6d01930179d
+Create Date: 2024-05-01 17:14:08.190904
+
+"""
+
+from alembic import op
+from sqlalchemy import orm
+from sqlalchemy import and_
+
+from dataall.core.environment.services.environment_service import EnvironmentService
+from dataall.core.permissions.services.resource_policy_service import ResourcePolicyService
+
+from dataall.modules.datasets_base.db.dataset_models import DatasetTable, Dataset
+from dataall.modules.dataset_sharing.db.share_object_models import ShareObject, ShareObjectItem
+from dataall.modules.dataset_sharing.services.dataset_sharing_enums import ShareItemStatus, ShareableType, PrincipalType
+
+# revision identifiers, used by Alembic.
+revision = '458572580709'
+down_revision = 'c6d01930179d'
```

Comment:
lets update with `a991ac7a85a2` as `down_revision` from migration script in PR https://github.com/data-dot-all/dataall/pull/1237

Reply:
migrations merged